src/internal/serviceenv/service_env.go

```diff
@@ -194,7 +194,7 @@ func (env *NonblockingServiceEnv) Config() *Configuration {
 }
 
 func (env *NonblockingServiceEnv) isWorker() bool {
-	return env.config.PPSPipelineName != ""
+	return env.config.PPSPipelineName != "" || env.config.IsPachw
```

Comment:
We cannot use the pipeline name field for this?

Reply:
We could, but since its not a pipeline, I felt like that would be hacky or misleading and opted to use a new config variable.

Reply:
Ohh, right. We want it to be considered a worker to avoid a direct connection to postgres. That makes sense.